pocs/tests/test_observatory.py

```diff
@@ -122,6 +164,12 @@ def test_primary_camera(observatory):
     assert observatory.primary_camera is not None
 
 
+def test_primary_camera_no_primary_camera(observatory):
+    obs = observatory
```

Comment:
This line is not necessary.

Did this test increase coverage of some line? I would be surprised if we weren't already testing that.

Reply:
> This line is not necessary.
> 
> Did this test increase coverage of some line? I would be surprised if we weren't already testing that.

I wrote this test because it wasn't tested. It was marked in red color.